dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/XtxtUML.xtext

```diff
@@ -212,10 +212,18 @@ TUMultiplicity :
 
 // Commons
 
+TUModifiers:
```

Comment:
Isn't this too complicated (and too strict, with regards to the order)? See rules [`CommonModifier`](https://github.com/eclipse/xtext-xtend/blob/314ca2fd50b88f8c585dd643c2234008a1148327/org.eclipse.xtend.core/src/org/eclipse/xtend/core/Xtend.xtext#L223-L226) and [`XtendTypeDeclaration`](https://github.com/eclipse/xtext-xtend/blob/314ca2fd50b88f8c585dd643c2234008a1148327/org.eclipse.xtend.core/src/org/eclipse/xtend/core/Xtend.xtext#L26) from the Xtend grammar.